### PR TITLE
[CORE-12592] Typeface Update

### PR DESCRIPTION
**Pull request by fweinrich**
[JIRA Link](https://jira.nebenan.de/browse/CORE-12592)

# Checklist

**Definition of Done**
- [x] PR approved
- [x] QA approved

**Road to Merge**
- [x] PR is **done**
- [x] Bring branch up to date with `development`
- [ ] **Squash and merge** (& delete branch)

### CODE REVIEW

**Comment by rejas** [comment]
And they told me that I was crazy when I updated it the last time :-D 

any particular reason / ticket you did this?

**Comment by spossner** [comment]
@florian where do we load the Inter font family? When infirmity for ads I found out, that product only provided true type font file..

**Comment by fweinrich** [comment]
@spossner  It was here https://github.com/good-hood-gmbh/nebenan-frontend/blob/5d5f66092ab8a15333a39f924556dbb2b51a6371/client/containers/home/components/layout/fonts.scss#L13  now moved to ui-library (was not needed for the initial PoC/Test)

File: fonts.scss [hunk 1]
```diff
@@ -1,7 +1,12 @@
...
-@import "@fontsource/open-sans/600";
+@import "@fontsource/inter/100";
+@import "@fontsource/inter/200";
+@import "@fontsource/inter/300";
+@import "@fontsource/inter/400";
+@import "@fontsource/inter/500";
+@import "@fontsource/inter/600";
+@import "@fontsource/inter/700";
+@import "@fontsource/inter/800";
+@import "@fontsource/inter/900";
```
Comment (spossner) [inline fonts.scss]: Do we need all font weights? Will that load all the woff files of all sizes? As far as I saw 400, 500, 600, 700, 800 are used upper in the file.. more somewhere else or potentially by design team randomly?

Comment (rejas) [reply fonts.scss]: will switch to the variable font in the next commits so no explicit font weight files then

**Pull request merged**